src/puppetlabs/ezbake/core.clj

```diff
@@ -99,14 +102,12 @@
 (defn relativize
   "Convert an absolute File to a relative File"
   [base-path absolute-file]
-  (let [base-file (if (instance? File base-path)
-                    base-path
-                    (File. base-path))]
```

Comment:
Ah, I was just trying to figure out how this function worked and in the process figured I'd clean it up a bit. Probably should have put it in a separate maint commit